helm/fluent-logshipping-app/values.yaml

```diff
@@ -23,10 +23,12 @@ fluentbit:
   groupID: 0
 
   image:
-    name: giantswarm/fluent-bit
+    name: giantswarm/fluent-bit-audited
     # -- Overrides the image tag whose default is the chart's appVersion
-    # This image fixes some systemd issues fixed in 1.8.x and adds the aws-for-fluent-bit cloudwatch plugin to be able to template cloudwatch log stream names.
-    tag: "1.9.8-aws-plugins"
```

Comment:
I guess the plugins we use are now included in the new image? And we changed the image name rather than having a custom tag?

Reply:
The image is a custom built image from a fork https://github.com/giantswarm/fluent-bit/blob/v0.1.0/dockerfiles/Dockerfile

I had to fork the repo instead of changing the image because fluent-bit is build without the exec plugin by default so I had to enable the exec plugin and add auditd in the container to be able to have ausearch in it